src/index.js

```diff
@@ -1,20 +1,32 @@
 import Button from './components/Button';
+import ConvertList from './components/ConvertList';
 import FilterList from './components/FilterList';
 import Header from './components/Header';
 import Modal from './components/Modal';
+import AddRestaurant from './components/ModalContents/AddRestaurant';
+import RestaurantDetails from './components/ModalContents/RestaurantDetails';
 import RestaurantBox from './components/RestaurantBox';
 import RestaurantInfo from './components/RestaurantInfo';
 import SelectList from './components/SelectList';
 import TextInput from './components/TextInput';
 import { restaurant } from './domain/restaurant';
 import './styles/index.css';
+import { $ } from './utils';
 
 restaurant.restaurants = JSON.parse(localStorage.getItem('restaurant') || '[]');
+restaurant.favoriteRestaurants = restaurant.restaurants.filter(
+  (restaurant) => restaurant.isFavorite
+);
```

Comment:
레스토랑의 멤버변수를 레스토랑의 함수를 이용해서 레스토랑 밖에서 수정하는건 좀 어색하지 않나요?

Reply:
```suggestion
restaurant.addFavorite();
```

함수를 만들어 놓고 활용을 하지 않았네요🥲 이렇게 호출하면 괜찮을까요 ?